templates/admin/user/list.tmpl

```diff
@@ -68,36 +68,33 @@
 						</th>
 						<th>{{.locale.Tr "email"}}</th>
 						<th>{{.locale.Tr "admin.users.activated"}}</th>
-						<th>{{.locale.Tr "admin.users.admin"}}</th>
 						<th>{{.locale.Tr "admin.users.restricted"}}</th>
-						<th>{{.locale.Tr "admin.users.2fa"}}</th>
```

Comment:
Me as a site admin who also does security work, I would really like to see whether a user is an admin and whether their account has setup 2FA

Reply:
Does it really have to be in the users table? If you say it's needed we can bring it back, it doesn't consume that much space....

Reply:
In my personal option, I prefer to have it in the users table if there is enough space.

While it is not a must, if most people prefer not.